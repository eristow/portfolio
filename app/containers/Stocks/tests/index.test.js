/**
 *
 * Tests for Stocks
 *
 * @see https://github.com/react-boilerplate/react-boilerplate/tree/master/docs/testing
 *
 */

import React from 'react';
import { render } from 'react-testing-library';
// import 'jest-dom/extend-expect'; // add some helpful assertions

import { Stocks } from '../index';

describe('<Stocks />', () => {
  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    const dispatch = jest.fn();
    render(<Stocks dispatch={dispatch} />);
    expect(spy).not.toHaveBeenCalled();
  });

  it('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(<Stocks />);
    expect(firstChild).toMatchSnapshot();
  });
});
