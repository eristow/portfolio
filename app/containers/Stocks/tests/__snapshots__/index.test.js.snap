// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stocks /> Should render and match the snapshot 1`] = `
<h1>
  Stock Calculator
</h1>
`;
